src/sdk-reference/js/6/getting-started/node-js/index.md

```diff
@@ -0,0 +1,210 @@
+</div>
+
+## Create your first "Hello World" document
+
+Create a `create.js` file with following code:
+
+[snippet=create]
+
+This code does the following:
+* creates a new document in `nyc-open-data` within the `yellow-taxi` index
+* logs a success message to the console if everything went fine
+* logs an error message if any of the previous actions failed
+* disconnects from Kuzzle after the document is created or if an error occurs
+
+Run your file with Node.js
+
+```bash
+node create.js
+```
+
+<div class="alert alert-success">
+You have now successfully stored your first document into Kuzzle. Click <a href="{{ site_base_path }}guide/1/essentials/installing-console">here</a> to see how you can use the
+   <a href="http://console.kuzzle.io" target="_blank"><strong>Kuzzle Admin Console</strong></a> to browse your collection and confirm that your document was saved.
+</div>
+
+<div class="alert alert-info">
+Having trouble? Get in touch with us on <a href="https://gitter.im/kuzzleio/kuzzle">Gitter!</a> We're happy to help.
+</div>
+
+_You can find more resources about Kuzzle Javascript SDK in the [SDK Reference]({{ site_base_path }}sdk-reference/js/6)._
+
+## Subscribe to realtime document notifications (pub/sub)
+
+Kuzzle provides pub/sub features that can be used to trigger real-time notifications based on the state of your data (for a deep-dive on notifications check out the <a href="{{ site_base_path }}sdk-reference/js/6/essentials/realtime-notifications/">realtime notifications</a> documentation).
+
+Let's get started. Create a `subscribe.js` file with following code:
+
+[snippet=subscribe]
+
+Run your file in Node.js
+
+```bash
+node subscribe.js
+```
+
+Your `subscribe.js` app is now running and monitoring any documents that match the filter, specifically documents that have a `license` field that is equals to `'B'`.
+
+Now in another terminal, launch the `create.js` file that we created in the previous section.
+
+```bash
+node create.js
+```
+
+This will create a new document in Kuzzle which will trigger a [document notification]({{ site_base_path }}api/1/essentials/notifications/#documents-changes-messages-default) in the `subscribe.js` app. Check the `subscribe.js` terminal to make sure a new log appears every time a document is created using the `create.js` app:
+
+```bash
+New driver Sirkis with id AWccRe3-DfukVhSzMdUo has B license.
+```
+
+<div class="alert alert-success">
+Congratulations! You have just choreographed your first pub/sub pattern!
+</div>
+
+<div class="alert alert-info">
+Having trouble? Get in touch with us on <a href="https://gitter.im/kuzzleio/kuzzle">Gitter!</a> We're happy to help.
+</div>
+
+## Where do we go from here?
+
+Now that you're more familiar with Kuzzle, dive even deeper to learn how to leverage its full capabilities:
+
+* take a look at the <a href="{{ site_base_path }}sdk-reference/js/6">SDK Reference</a>
```

Comment:
That link redirects... to this same page, since this is the landing of the JS SDK.

```suggestion
* discover what this SDK has to offer by browsing other sections of this documentation
```